src/render-instruction.ts

```diff
@@ -370,6 +385,7 @@ ${accountsType}
     ${accountsParamDoc}${createInstructionArgsComment}
      * @category Instructions
      * @category ${this.upperCamelIxName}
+     * @category ${this.upperCamelIxName}
```

Comment:
This appears to be a mistake (same category twice)?

Reply:
good catch. Must have accidently duplicated the line. Not sure why I didnt catch that.